R/shinychat-package.R

```diff
@@ -5,9 +5,14 @@
 #' @importFrom coro async
 #' @importFrom htmltools tag css HTML
 #' @import rlang
+#' @import S7
 ## usethis namespace: end
 NULL
 
+# enable usage of <S7_object>@name in package code
+#' @rawNamespace if (getRversion() < "4.3.0") importFrom("S7", "@")
```

Comment:
Is this because it just comes with R now?

Reply:
Yes? It is from S7's articles: https://rconsortium.github.io/S7/articles/packages.html#backward-compatibility

Reply:
Thanks!